### DISCUSSION TITLE
Docker command not working as in example

### DISCUSSION THREAD

**Discussion by sudipstha08** [vercel/next.js#35960] [Help]
### Verify canary release

- [X] I verified that the issue exists in Next.js canary release

### Provide environment information

Operating System:
      Platform: linux
      Arch: x64
      Version: #44~20.04.1-Ubuntu SMP Thu Mar 24 16:43:35 UTC 2022
    Binaries:
      Node: 14.18.1
      npm: 8.5.5
      Yarn: 1.22.17
      pnpm: N/A
    Relevant packages:
      next: 12.1.4
      react: 17.0.2
      react-dom: 17.0.2

### Describe the Bug

This command is not working 
```
docker build -t nextjs-docker .
```

I did this
```
docker build . -t nextjs-docker
```

### Expected Behavior

Run docker example successfully after following the steps provided

### To Reproduce

Try the docker example

**Comment by David-Odhiambo** [comment]
docker build -t nextjs-docker.
